chatops-settings.yaml

```diff
@@ -0,0 +1,140 @@
+
+
+
+
+
+
+
+export ST2_HOSTNAME="${ST2_HOSTNAME:-localhost}"
```

Comment:
Looks like I forgot to remove it. Will do that now since we don't need it anymore